services/movie.py

```diff
@@ -6,16 +7,18 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
```

Comment:
Use Optional type hint, it indicates that a variable can be either of the specified type or None

Correct in all similar cases